docs/references/config/config.toml.md

```diff
@@ -0,0 +1,1749 @@
+```toml
+timeout_prevote = "1s"
+```
+
+| Value type          | string (duration) |
+|:--------------------|:------------------|
+| **Possible values** | &gt;= `"0s"`      |
+
+### consensus.timeout_prevote_delta
+How much the timeout_prevote increases with each round.
+```toml
+timeout_prevote_delta = "500ms"
+```
+
+| Value type          | string (duration) |
+|:--------------------|:------------------|
+| **Possible values** | &gt;= `"0ms"`     |
+
+### consensus.timeout_precommit
+How long we wait after receiving +2/3 precommits for “anything” (ie. not a single block or nil).
+```toml
+timeout_precommit = "1s"
+```
+
+| Value type          | string (duration) |
+|:--------------------|:------------------|
+| **Possible values** | &gt;= `"0s"`      |
+
+### consensus.timeout_precommit_delta
+How much the timeout_precommit increases with each round.
+```toml
+timeout_precommit_delta = "500ms"
+```
+
+| Value type          | string (duration) |
+|:--------------------|:------------------|
+| **Possible values** | &gt;= `"0ms"`     |
+
+### consensus.timeout_commit
+How long we wait after committing a block, before starting on the new height.
+This gives us a chance to receive some more precommits, even though we already have +2/3.
+```toml
+timeout_commit = "1s"
+```
+
+| Value type          | string (duration) |
+|:--------------------|:------------------|
+| **Possible values** | &gt;= `"0s"`      |
+
+### consensus.double_sign_check_height
+How many blocks to look back to check existence of the node's consensus votes before joining consensus.
+```toml
+double_sign_check_height = 0
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt;= 0 |
+
+When non-zero, the node will panic upon restart if the same consensus key was used to sign {double_sign_check_height}
+last blocks. So, validators should stop the state machine, wait for some blocks, and then restart the state machine to
+avoid panic.
+
+### consensus.skip_timeout_commit
+Make progress as soon as we have all the precommits.
+```toml
+skip_timeout_commit = false
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `false` |
+|                     | `true`  |
+
+This is similar to setting [`consensus.timeout_commit`](#consensustimeout_commit) to `"0s"`.
+
+### consensus.create_empty_blocks
+If there are no transactions in the mempool, empty blocks are proposed to indicate that the chain is still running.
+```toml
+create_empty_blocks = true
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `true`  |
+|                     | `false` |
+
+This is more relevant to networks with a low number of transactions.
+
+### consensus.create_empty_blocks_interval
+If there are no transactions in the mempool, empty blocks are proposed to indicate that the chain is still running at
+this interval.
+```toml
+create_empty_blocks_interval = "0s"
```

Comment:
Does 0 mean wait forever?

Reply:
seems so based on code logic and information below